src/components/portfolioComponents/portfolioGraph/portfoliograph.js

```diff
@@ -0,0 +1,68 @@
+import { ResponsiveContainer, LineChart, CartesianGrid, XAxis, YAxis, Tooltip, Line } from "recharts";
+import { useState, useEffect } from "react";
+import { Container } from "react-bootstrap"
+
+function PortfolioGraph({ data }) {
+    const [tickBoolean, setTickBoolean] = useState(false)
+
+    window.addEventListener("resize", showTick);
+
+    function showTick() {
+        if (window.innerWidth >= 576) {
+            setTickBoolean(true)
+        }
+        else {
+            setTickBoolean(false)
+        }
+    }
+    useEffect(() => {
+        showTick()
+    }, [])
+
+
+    const CustomToolTip = props => {
```

Comment:
Break this tooltip out into its own component under ```widgets``` and require it. You can save yourself work here, the tooltip can then be reused for the individual stock page graph, comparison graph, and any other graphs which have axis of "Date" and "Price".